middle_end/flambda/types/template/flambda_type.templ.ml

```diff
@@ -869,6 +869,9 @@ let reify ?allowed_if_free_vars_defined_in ?additional_free_var_criterion
         if is_bottom env imms then
           begin match Row_like.For_blocks.get_singleton blocks with
           | None -> try_canonical_simple ()
+          | Some ((tag, _), _) when Tag.(equal double_array_tag tag) ->
+            (* CR do better, and be more careful in the next case *)
```

Comment:
Please put a username next to "CR" e.g. "CR gbury".  Also can you add more detail about what needs improving in the next case?

Reply:
Comment updated. For one thing, in the next case, one could ensure that the tag is indeed scannable (which would have caught this error in a more informative way).